layouts/index.html

```diff
@@ -47,15 +47,15 @@ <h1 class="mt-0"><img src="/images/localstack.svg" alt="LocalStack Logo" style="
         <span style="font-size: 0.875rem; font-weight: 700">CONTRIBUTORS</span>
       </div>
       <div class="col-12 col-sm-6 col-lg-3">
-        <span style="font-size: 2.5rem; font-weight: 800">20 k<span style="color: #79F0A9">+</span></span><br>
```

Comment:
I think we've agreed with @HarshCasper  to keep this number at 20k for now, as not all invited Slack members are in fact active. This way we're more staying on the conservative/realistic side, and make sure the numbers are not exaggerated 👍

Reply:
makes sense, done :+1: